lib/datasource/remote/api/network_repository.dart

```diff
@@ -5,7 +5,7 @@ import 'package:seeds/datasource/remote/util/response_extension.dart';
 
 abstract class NetworkRepository {
   String baseURL = remoteConfigurations.defaultEndPointUrl;
-  String v2historyURL = remoteConfigurations.defaultV2EndPointUrl;
+  String v2historyURL = remoteConfigurations.defaultEndPointUrl;
```

Comment:
Can't do that, please keep v2 history separate from non-V2, that's why we have it

Reply:
added it back